src/lib/viewers/media/MediaBaseViewer.js

```diff
@@ -44,7 +44,6 @@ class MediaBaseViewer extends BaseViewer {
         this.toggleMute = this.toggleMute.bind(this);
         this.handleRate = this.handleRate.bind(this);
         this.handleAutoplay = this.handleAutoplay.bind(this);
-        this.mediaendHandler = this.mediaendHandler.bind(this);
```

Comment:
I think leaving in this event is still fine, we just don't need to react to it internally in the preview sdk

Reply:
I see